packages/daemon/src/types.d.ts

```diff
@@ -725,6 +726,9 @@ export type DeferredTasks<T extends Record<string, string | string[]>> = {
 
 type FormulateNumberedGuestParams = {
   guestFormulaNumber: string;
+  handleId: string;
+  guestId: string;
```

Comment:
For consistency with `hostAgentId` and `hostHandleId` in the same interface:
```suggestion
  guestHandleId: string;
  guestAgentId: string;
```

Would also accept `agentId` / `handleId`.

Reply:
I would like to keep these as present, or instead `ownHandleId` and `ownAgentId`. The role of `handleId` and `agentId` is the same between Host and Guest so I would like to use the same names. Whereas a guest has a host, but a host does not. (I would imagine adding `PARENT` for hosts created by other hosts. If we explored that path, we’d have to consider how to accommodate the specialness of the root host. We could borrow the `/..` == `/` trick from *ix file systems.)

Reply:
That makes sense. I'd prefer the more explicit `ownHandleId` / `ownAgentId` in that case, but I leave it up to you.

Reply:
Sure. I will circle back for this.